test/internal/testenv/testenv.go

```diff
@@ -92,6 +92,13 @@ func ControllerFeatureGates() string {
 	return featureGates
 }
 
+// ExpressionRoutesEnabled indicates whether or not to enable expression routes
+// for the Kong Gateway and the controller.
+// If none specified, we fall back to default values.
```

Comment:
This last sentence is a bit vague. What does it mean we "fall back to default values"? Meaning default router? What is it then? 

We could mention that as of now the default router is X (is it `traditional_compatible`?)